vendor/engines/secure_rooms/app/views/secure_rooms/indala_numbers/edit.html.haml

```diff
@@ -0,0 +1,17 @@
+= content_for :head_content do
+  = render "users/form_head"
+
+= content_for :h1 do
```

Comment:
Not sure why it's not showing, but the tabnav (Details, Orders, etc) is disappearing in this view.